tests/test_integration.py

```diff
@@ -115,7 +115,7 @@
     }
     wildtype = await restore_model('iJO1366')
     modified = await modify_model(message, wildtype.copy())
-    assert len(modified.medium) == len(message['medium'])
+    assert len(modified.medium) + 1 == len(message['medium'])
```

Comment:
Suggest instead of hardcoding this instead do something like:

    assert len(modified.medium) + len(MediumChangeModel.TRACE_METALS) == len(message['medium'])